mobile/src/main/AndroidManifest.xml

```diff
@@ -58,7 +58,12 @@
          </activity>
 
         <activity
-            android:name=".ui.MainActivity" />
+            android:name=".ui.MainActivity" >
+            <intent-filter>
+                <action android:name="com.google.samples.apps.iosched.STARTUP_ACTIVITY" />
```

Comment:
I have 0 experience with AppStartup, but, can this be applied to the LauncherActivity to take its performance loss into account?

Reply:
This is not related to `AppStartup` rather `Startup Macrobenchmark` library. So I measured `MainActivity`.